sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1144,7 +1145,7 @@ class BlobProperties(DictMixin):
     """The container in which the blob resides."""
     snapshot: Optional[str]
     """Datetime value that uniquely identifies the blob snapshot."""
-    blob_type: "BlobType"
+    blob_type: Optional["BlobType"]
```

Comment:
So I think we already released all the changes to this on class when someone requested it? If those types were wrong we may get some complaints and need to patch this in main too.

Reply:
Yeap. 😞 
```
azure\storage\blob\_deserialize.py:159: error: Incompatible types in assignment (expression has type "Optional[BlobType]", variable has type "BlobType")  [assignment]
azure\storage\blob\_deserialize.py:169: error: Incompatible types in assignment (expression has type "Optional[datetime]", variable has type "datetime")  [assignment]
azure\storage\blob\_deserialize.py:171: error: Incompatible types in assignment (expression has type "Optional[int]", variable has type "int")  [assignment]
azure\storage\blob\_deserialize.py:173: error: Incompatible types in assignment (expression has type "Optional[bool]", variable has type "bool")  [assignment]
azure\storage\blob\_deserialize.py:177: error: Incompatible types in assignment (expression has type "Union[str, AccessTier, None]", variable has type "Optional[StandardBlobTier]")  [assignment]
```

And these should now be correct since these are raised from [get_blob_properties_from_generated_code](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/azure/storage/blob/_deserialize.py#L134)

Reply:
So after our discussion yesterday, I think that I will revert this file for now and blissfully ignore any errors this stuff raises until I get to the PR containing `_models`.